setup.sh

```diff
@@ -15,6 +15,19 @@ sudo apt-get install -y postgresql postgresql-contrib
 echo "install requirements"
 pip3 install -r /vagrant/requirements.txt
 
-echo "running flask_init.py"
-export FLASK_APP=/vagrant/flask_init.py
-python3 -m flask run --host=0.0.0.0 >> /vagrant/log.log 2>&1 &
+echo "configuring database"
+sudo su postgres <<POSTGRESQL_COMMANDS
+psql
+CREATE DATABASE edison;
+ALTER ROLE postgres WITH PASSWORD 'edison';
+POSTGRESQL_COMMANDS
+
+export FLASK_ENV=development
+
+echo "running flask_serve_static.py"
+export FLASK_APP=/vagrant/flask_serve_static.py
+flask run -h 0.0.0.0 -p 5000 >> /vagrant/flask_serve_static.log 2>&1 &
+
+echo "running flask_restful_api.py"
+export FLASK_APP=/vagrant/backend/flask_restful_api.py
+flask run -h 0.0.0.0 -p 3000 >> /vagrant/flask_restful_api.log 2>&1 &
```

Comment:
Probably a good idea to set 5000 and 3000 as global variables and put them at the top of the file, so that it is obvious what ports are for what

Reply:
Ack.